concepticondata/conceptlists/Marchese-1983-311.tsv

```diff
@@ -0,0 +1,312 @@
+ID	NUMBER	FRENCH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	Category
+Marchese-1983-311-1	1	abeille	665	BEE	bee	LES ANIMAUX
+Marchese-1983-311-2	2	agouti	313	AGOUTI	agouti	LES ANIMAUX
+Marchese-1983-311-3	3	âne	1862	DONKEY	donkey	LES ANIMAUX
+Marchese-1983-311-4	4	animal	619	ANIMAL	animal	LES ANIMAUX
+Marchese-1983-311-5	5	araignée	843	SPIDER	spider	LES ANIMAUX
+Marchese-1983-311-6	6	buffle	934	BUFFALO	buffalo	LES ANIMAUX
+Marchese-1983-311-7	7	caméléon	1555	CHAMELEON	chameleon	LES ANIMAUX
+Marchese-1983-311-8	8	canard	1360	DUCK	duck	LES ANIMAUX
+Marchese-1983-311-9	9	carpe	2685	CARP (FISH)	carp	LES ANIMAUX
+Marchese-1983-311-10	10	chat	1208	CAT	cat	LES ANIMAUX
+Marchese-1983-311-11	11	chauve-souris	1793	BAT	bat	LES ANIMAUX
+Marchese-1983-311-12	12	cheval	615	HORSE	horse	LES ANIMAUX
+Marchese-1983-311-13	13	chèvre	1502	GOAT	goat	LES ANIMAUX
+Marchese-1983-311-14	14	chien	2009	DOG	dog	LES ANIMAUX
+Marchese-1983-311-15	15	coq	1511	ROOSTER	rooster	LES ANIMAUX
+Marchese-1983-311-16	16	crabe	2348	CRAB	crab	LES ANIMAUX
+Marchese-1983-311-17	17	crocodile	1857	CROCODILE	crocodile	LES ANIMAUX
+Marchese-1983-311-18	18	éléphant	1290	ELEPHANT	elephant	LES ANIMAUX
+Marchese-1983-311-19	19	escargot	1543	SNAIL	snail	LES ANIMAUX
+Marchese-1983-311-20	20	fourmi	587	ANT	ant	LES ANIMAUX
+Marchese-1983-311-21	21	fourmi magnan			magnan ant	LES ANIMAUX
+Marchese-1983-311-22	22	guêpe-maçonne			mason wasp	LES ANIMAUX
+Marchese-1983-311-23	23	hippopotame	3145	HIPPOPOTAMUS	hippopotamus	LES ANIMAUX
+Marchese-1983-311-24	24	margouillat			common agama	LES ANIMAUX
```

Comment:
Based on a Google picture search this could be maybe mapped to 2355 GECKO.

Reply:
Yes, Pons also gives this translation.